### PR TITLE
OD-590 [Feature] Added translation to Widget-ChartPie

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@sofiiakvasnevska @inna-bieshulia

OD-590 https://weboo.atlassian.net/browse/OD-590

## Description
Added user translations to widget - ChartPie

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/build.js [hunk 1]
```diff
@@ -97,14 +104,15 @@
...
-                    name: row[data.dataSourceQuery.columns.category] || 'Category ' + (i + 1),
+                    name: row[data.dataSourceQuery.columns.category] || T('widgets.chart.pie.category') + ' ' + TN(i + 1),
                     y: parseInt(row[data.dataSourceQuery.columns.value], 10) || 0
                   });
                 });
                 break;
               case 1:
-                // Summarize data
-                data.name = 'Count of ' + data.dataSourceQuery.columns.column;
+                // Summarise data
```
Comment (tonytlwu) [inline js/build.js]:
```suggestion
                // Summarize data
```

File: js/build.js [hunk 2]
```diff
@@ -8,7 +8,13 @@
     Fliplet.Widget.instance('chart-pie-1-1-0', function(data) {
       var chartId = data.id;
       var $container = $(this);
-      var themeInstance = Fliplet.Themes.Current.getInstance();
+      // var themeInstance = Fliplet.Themes.Current.getInstance();
+      var themeInstance = {
```
Comment (tonytlwu) [inline js/build.js]: Why is this here?

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/build.js (hunk 2 above)
Comment (OleksandrLavrynenko) [reply js/build.js]: Sorry. This code was used for the test

**Review by tonytlwu — COMMENTED** [review]
@romanyosyfiv @inna-bieshulia Translation looks ready. This can be tested once API is ready.

**Pull request closed without merging**